selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/provider/TestWorkspaceAgentApiEndpoint.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.selenium.core.provider;
+
+/** Workspace agent API endpoint */
+public interface TestWorkspaceAgentApiEndpoint {
+
+  /**
+   * get workspace agent API endpoint
+   *
+   * @param workspaceId
+   * @return
+   * @throws Exception
+   */
+  String get(String workspaceId) throws Exception;
```

Comment:
It's more safe to return URL type, so as this method returns URL to workspace agent.